tools/apiview/parsers/rust-api-parser/clean-rust-doc-output/src/main.rs

```diff
@@ -0,0 +1,37 @@
+use serde_json::Value;
+use std::fs::File;
+use std::io::{self, Read, Write};
+
+fn main() -> io::Result<()> {
+    // Read the JSON file generated by rustdoc
+    let mut file = File::open("inputs/docs.json")?;
+    let mut data = String::new();
+    file.read_to_string(&mut data)?;
+
+    // Parse the JSON data
+    let v: Value = serde_json::from_str(&data).expect("Unable to parse JSON");
```

Comment:
Case in point: it's weird using `?` and `expect()`, which panics. If you use the recommendation above, use `?` here and everywhere else as well.